.github/workflows/test.yml

```
@@ -0,0 +1,72 @@
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_dispatch: {}
+
+name: Test
+
+jobs:
+  host_tests:
+    strategy:
+      matrix:
+        # os: [macos-latest, windows-2019]
+        os: ['ubuntu-latest']
+        node: [16.x, 18.x, 20.x, 22.x]
```

```suggestion
        node: [16.20.1, 18.x, 20.x, 22.x]
```
Should we pin the 16 so we check our minimum?

Or we should at least run one CI test on that exact version before we call the package compatible (so we can defer this to when the implementation is done, or we can pin now and unpin later.)